main/src/propagator/factory.hpp

```diff
@@ -59,6 +62,24 @@ propagatorFactory(const std::string& choice, size_t ngmax, size_t ng0, std::ostr
         return std::make_unique<TurbVeProp<DomainType, ParticleDataType>>(ngmax, ng0, output, rank);
 #endif
     }
+    if (choice == "std-net14")
+    {
+#ifdef USE_NUCLEAR_NETWORKS
+        return std::make_unique<NuclearProp<DomainType, ParticleDataType, 14, false>>(ngmax, ng0, output, rank);
+#endif
+    }
+    if (choice == "std-net86")
+    {
+#ifdef USE_NUCLEAR_NETWORKS
+        return std::make_unique<NuclearProp<DomainType, ParticleDataType, 86, false>>(ngmax, ng0, output, rank);
+#endif
+    }
+    if (choice == "std-net87")
+    {
+#ifdef USE_NUCLEAR_NETWORKS
+        return std::make_unique<NuclearProp<DomainType, ParticleDataType, 87, false>>(ngmax, ng0, output, rank);
+#endif
+    }
```

Comment:
Coalescing the macro into a single block would probably look nicer visually.

Reply:
Ok I will move to a single ifdef if.